### PR TITLE
Release v0.34.0

### PR DESCRIPTION
**Pull request by nghialv**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Review by knanao — DISMISSED** [review]
🚀

File: docs/content/en/blog/releases/v0.34.0.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+date: 2022-07-05
+description: >
+ Release v0.34.0
+---
+
+## Release v0.34.0 with changes since v0.33.0
+
+### Notable Changes
+
+* Add the ability to disable/enable and delete application on the application detail page ([#3773](https://github.com/pipe-cd/pipecd/pull/3773))
```
Comment (khanhtc1202) [inline docs/content/en/blog/releases/v0.34.0.md:13]:
```suggestion
* Add actions menu to the application detail page ([#3773](https://github.com/pipe-cd/pipecd/pull/3773) & [#3776](https://github.com/pipe-cd/pipecd/pull/3776))
```
how about this?

**Review by khanhtc1202 — APPROVED** [review]
🙌

**Review by khanhtc1202 — APPROVED** [review]
🚀

**Review by knanao — APPROVED** [review]

**Pull request merged**